python/ray/rllib/agents/impala/vtrace_policy.py

```diff
@@ -258,32 +256,13 @@ def make_time_major(tensor, drop_last=False):
             rewards=make_time_major(rewards, drop_last=True),
             values=make_time_major(values, drop_last=True),
             bootstrap_value=make_time_major(values)[-1],
-            dist_class=dist_class,
+            dist_class=Categorical if is_multidiscrete else dist_class,
             valid_mask=make_time_major(mask, drop_last=True),
             vf_loss_coeff=self.config["vf_loss_coeff"],
             entropy_coeff=self.config["entropy_coeff"],
             clip_rho_threshold=self.config["vtrace_clip_rho_threshold"],
             clip_pg_rho_threshold=self.config["vtrace_clip_pg_rho_threshold"])
 
-        # KL divergence between worker and learner logits for debugging
-        model_dist = MultiCategorical(unpacked_outputs)
```

Comment:
Removed the KL stats since I doubt they were very useful

Reply:
I think they should stay in. They can be useful for debugging the policy (e.g. large fluctuations are a bad sign). See slide 21 [here](http://rll.berkeley.edu/deeprlcourse/docs/nuts-and-bolts.pdf).